src/main/java/core/basesyntax/csv/writer/WriterService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.csv.writer;
+
+import core.basesyntax.model.Fruit;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public interface WriterService {
+    void writeToFile(FileWriter fileWriter, List<Fruit> fruitList) throws IOException;
+}
```

Comment:
this is not quite good that our writer needs some ready FileWriter

also, we should be able to write any data, not only Fruits